discussions-server/src/main/java/org/xwiki/contrib/discussions/internal/server/DiscussionsResourceReferenceHandler.java

```diff
@@ -207,6 +214,23 @@ public void handle(ResourceReference reference, ResourceReferenceHandlerChain ch
         handleNext(reference, chain);
     }
 
+    private void checkWiki(DiscussionsResourceReference discussionsResourceReference)
+        throws ResourceReferenceHandlerException
+    {
+        if (discussionsResourceReference.getWikiReference() != null) {
```

Comment:
No error when the wiki reference is missing?

Reply:
I will double check but AFAIR we added later the support of wiki references there and so we accept it to be null for backward compatibility reasons.

Reply:
You can actually see line 179 that the if already existed